modules/ROOT/pages/metadata-output.adoc

```diff
@@ -11,6 +11,8 @@ isolated from each other.
 
 == Declaring an OutputTypeResolver and AttributesTypeResolver
 
+An `OutputTypeResolver` returns a `MetadataType` that describes the class returned by an operation.
```

Comment:
I don't quite agree with this edit and its wording, specially in the context of Java code. A class (OutputTypeResolver) cannot return an object (MetadataType), it is the class methods that return objects (in this case the getOutputType method). I think the implementation of OutputTypeResolver is clear by looking below at the description + java code. Also instead of 'describes' I'd say it assigns a MetadataType (which is mule language) to a class (java language).
Plus it is not limited only for operations, it can be used in other components too (like sources).
What was the customer feedback? Was it about the Java implementation? Or was it about the functionality / what does this feature do (which I guess can be improved a bit)?